apple/Sources/Sargon/Extensions/Methods/Crypto/Keys/Ed25519PublicKey+Wrap+Functions.swift

```diff
@@ -12,6 +12,14 @@ extension Ed25519PublicKey {
         // Rust: `new_ed25519_public_key_from_bytes`
         self = try newEd25519PublicKeyFromBytes(bytes: Data(bytes))
     }
+    
+    public init(jsonData: some DataProtocol) throws {
+        self = try newEd25519PublicKeyFromBytes(bytes: Data(jsonData))
```

Comment:
@danvleju-rdx this is semantically incorrect and will break if we were to change the JSON representation of the keys. Instead you should do:

```rust
json_data_convertible!(Ed25519PublicKey);
```

in Rust, and then use the newly exported UniFFI methods which will be:
`newEd25519PublicKeyFromJsonBytes` (N.B. the `Json`.)
and 
`ed25519PublicKeyToJsonBytes`